packages/shared/lib/core/layer-2/errors/layer1-recipient.error.ts

```diff
@@ -0,0 +1,14 @@
+import { BaseError } from '@core/error/classes'
+import { localize } from '@core/i18n'
+
+export class Layer1RecipientError extends BaseError {
+    constructor() {
+        const message = localize('error.layer2.layer1Recipient')
+        super({
+            message,
+            showNotification: false,
+            saveToErrorLog: false,
+            logToConsole: true,
+        })
+    }
+}
```

Comment:
why do we need this error class?

Reply:
Ideally we would have a lot of error classes like this. I added this one specific to remove the duplication here: https://github.com/iotaledger/firefly/blob/38a1c05432452f76ca56bc2f7c83fe1cf8db033b/packages/shared/lib/core/utils/crypto/utils/validateEthereumAddress.ts#L46